packages/react/src/components/Storage/StorageManager/hooks/useStorageManager/useStorageManager.tsx

```diff
@@ -39,8 +39,8 @@ export function useStorageManager(
     files: defaultFiles.map((file) => {
       return {
         ...file,
-        id: file.s3Key,
-        name: file.s3Key,
+        id: file.key,
+        key: file.key,
```

Comment:
If `id` and `key` resolve to the same value do we need both?

Reply:
good question

Reply:
While they are the same value, my thinking was that I didn't want to conflate the `id` we use internally with the `key` for the file. But I'm also ok with merging them.

Reply:
Merging would make the most sense imo, concern is that it might be confusing to understand why we have both in the future

Reply:
We found an edge case where the customers can update the desired key (onProcessFile), which would mutate the id if they are merged. I think we should keep them separate even if they are often the same.

Reply:
Makes sense, can we add a comment here?